packages/design-system/src/components/Button/Button.tsx

```
@@ -16,29 +23,40 @@ const getVariantClass = ({
 }) => {
   if (disabled) {
-    return "bg-gray-400 text-inverse";
+    return "bg-gray-400 text-inverse border-transparent";
   }
 
   switch (variant) {
-  case "primary":
-    return pressed
-      ? "bg-primary-button text-inverse"
-      : "bg-primary text-inverse hover:bg-primary-button active:bg-primary-button";
+    case "primary":
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)